### DISCUSSION TITLE
Confluent Kafka Configuration

### DISCUSSION THREAD

**Discussion by konigbach** [roadrunner-server/roadrunner#1849] [Q&A] [answered]
Hi,
is it possible to add this configuration:
```
security.protocol=SASL_SSL
```
I haven't seen it here: https://roadrunner.dev/docs/queues-kafka/current/en
Thanks!

**Comment by rustatian** [comment, marked as answer]
Hey @konigbach 👋 
SSL is not supported at the moment. The first step to support this option is to create a feature request explaining this feature from the Kafka and user sides (use cases, etc). Then I'll plan it, implement and release 😃

**Reply by konigbach** [reply]
Thank you!

**Reply by TheLevti** [reply]
It's a standard feature across the board when connecting to any dependency (database, caches, etc.). I think the requirement for SSL is pretty much self explanatory.

Using kafka in the cloud with SSL enable (AWS MSK or Confluent Cloud), is kind of the industry standard.

The underlying library already supports this (rdkafka), so I don't see it as a big issue to enable it to be used.

For us this is a big blocker to continue use rr/spiral in the way how the framework architecture intends us to use it. We would greatly appreciate it if this missing part could land on the roadmap.

**Reply by rustatian** [reply]
@TheLevti I have nothing against supporting TLS 😃 Community is already working on that: https://github.com/roadrunner-server/kafka/pull/343

**Reply by TheLevti** [reply]
Thank you, did not see this. 🥳
